py-polars/tests/test_df.py

```diff
@@ -1697,9 +1697,6 @@ def test_is_unique() -> None:
 def test_sample() -> None:
     df = pl.DataFrame({"foo": [1, 2, 3], "bar": [6, 7, 8], "ham": ["a", "b", "c"]})
 
-    # by default samples should be random
```

Comment:
Ah, I see. By chance they can be equal. :sweat_smile:

Reply:
Exactly. Depending on the machine the system clock would be used to seed too. So I wonder if that's what did it or if it was just pure luck.

Reply:
Either way random != different every time. So there isn't really a great test for the behavior I think.